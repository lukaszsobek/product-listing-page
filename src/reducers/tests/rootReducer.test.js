import rootReducer from "../";

describe("rootReducer returns",() => {

    const state = {
        activeFilters: { categories: ["test"], colors: ["test2"] },
        products: [
            { colors: ["blue"], category: "one" },
            { colors: ["green"], category: "two" },
        ]
    };

    const action = {}

    it("default state",() => {
        expect(rootReducer(state, action)).toMatchSnapshot();
    });


});