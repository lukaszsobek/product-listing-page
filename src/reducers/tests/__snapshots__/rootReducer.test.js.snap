// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rootReducer returns default state 1`] = `
Object {
  "activeFilters": Object {
    "categories": Array [
      "test",
    ],
    "colors": Array [
      "test2",
    ],
  },
  "filteredProducts": Array [
    Object {
      "category": "one",
      "colors": Array [
        "blue",
      ],
    },
    Object {
      "category": "two",
      "colors": Array [
        "green",
      ],
    },
  ],
  "products": Array [
    Object {
      "category": "one",
      "colors": Array [
        "blue",
      ],
    },
    Object {
      "category": "two",
      "colors": Array [
        "green",
      ],
    },
  ],
}
`;
